workflow/doc.go

```diff
@@ -393,13 +393,13 @@ SideEffect function any other way than through its recorded return value.
 
 Query API
 
-A workflow execution could be stuck at some state for longer than expected period. Cadence provide facilities to query
-the current call stack of a workflow execution. You can use cadence-cli to do the query, for example:
+A workflow execution could be stuck at some state for longer than expected period. Temporal provide facilities to query
+the current call stack of a workflow execution. You can use temporal-cli to do the query, for example:
 
-	cadence-cli --domain samples-domain workflow query -w my_workflow_id -r my_run_id -qt __stack_trace
+	temporal-cli --domain samples-domain workflow query -w my_workflow_id -r my_run_id -qt __stack_trace
```

Comment:
![image](https://user-images.githubusercontent.com/2232524/75907563-602ea480-5dfd-11ea-92a4-c6e0e5e3b0a3.png)

Reply:
I mean CLI will be `temporal` not `temporal-cli`. Actually I believe it needs to be renamed to `tctl`.

Reply:
Let's iterate over this as separate PR.